crates/matrix-sdk-ui/CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased] - ReleaseDate
 
 ### Features
-
+- [**breaking**] Space rooms are now being retrieved through sliding sync and the newly introduced 
+  `NonSpace` filter should be used to exclude them from any room list.
```

Comment:
The filter name is `new_filter_non_space`. Ideally, you can prefix with `room_list_service::filters::new_filter_non_space` to indicate it's about the RoomList service, because so far the changelog entry doesn't mention it.